src/services/railcontent.js

```diff
@@ -343,17 +343,17 @@ export async function fetchHandler(url, method = 'get', dataVersion = null, body
 }
 
 export async function fetchUserLikes(currentVersion) {
-  let url = `/content/user/likes/all`
+  let url = `/content/v1/user/likes/all`
```

Comment:
Routes go through a different path now. 
Was this tested?
```suggestion
  let url = `/content/v1/user/likes`
```